src/data/url.test.tsx

```diff
@@ -28,3 +28,36 @@ describe("normalizeLink", () => {
     expect(normalizeLink("<test>")).toBe("<test>");
   });
 });
+
+describe("isLocalLink", () => {
+  const OLD_LOCATION = window.location;
+  beforeEach(() => {
+    Object.defineProperty(window, "location", {
+      value: new URL("https://excalidraw.com/"),
+      writable: true,
+    });
+  });
+  afterAll(() => {
+    Object.defineProperty(window, "location", {
+      value: OLD_LOCATION,
+      writable: true,
+    });
+  });
+  it("should return true for local links", () => {
+    expect(isLocalLink("./test")).toBe(true);
```

Comment:
this shouldn't pass as the idea here is to allow HTTP only urls. Though it may have a use case (/cc @zsviczian Obsidian?), we'd have to review usage in the codebase and whether it would work.